### DISCUSSION TITLE
New Video - Roadmap Update Q3/2023

### DISCUSSION THREAD

**Discussion by cagataycivici** [primefaces/community#31] [Announcements]
Hi all,

So much going on at PrimeLand, to give an update we've created a [new video](https://www.youtube.com/watch?v=XxNsa4YB_RA) to go over these in summary.

![image](https://github.com/primefaces/community/assets/686247/729f992e-2f96-4d27-9185-ef3e696265de)

Always bet on Prime!

**Comment by ibrahimkelly** [comment]
Awesome

**Comment by FrankGi1** [comment]
Is great to see PT and un-styled features !!! this adds up to going big with the flexibility; This one of the reason we have chosen prime (react) components to work with ! 

Keep up the great work !!!

**Reply by cagataycivici** [reply]
Awesome to hear! Unstyled PrimeReact is getting ready for launch next week 🚀
